application/audit/services/chapter_review_service.py

```
@@ -152,6 +194,131 @@ async def review_chapter(self, novel_id: str, chapter_number: int) -> ChapterRev
+                    location=f"Chapter {chapter.chapter_number}",
+                    suggestion=drift.get("suggestion")
+                ))
+        else:
+            logger.warning(f"Style consistency check JSON parse failed: {errs}")
+
+        return issues
+
+    def _build_style_consistency_prompt(self, chapter_content: str) -> str:
+        return f"""请检查以下章节内容的文风与叙事节奏是否存在偏移或异常。
+
+章节内容：
```

Ruff (0.15.10)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)